File: Backend/tranvel/src/main/java/com/ssafy/tranvel/dto/AnnouncementDto.java
package com.ssafy.tranvel.dto;

import jakarta.validation.constraints.NotEmpty;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter @Setter
public class AnnouncementDto {

    @NotEmpty
    private String title;

    @NotEmpty
    private String content;

}

File: Frontend/app/src/main/java/com/ssafy/tranvel/data/model/dto/AnnouncementDto.kt
package com.ssafy.tranvel.data.model.dto

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class AnnouncementDto(
    val id : Long?,
    val title : String?,
    val content : String?,
    val dateTime : String?
) : Parcelable{
    companion object{
        fun init() = AnnouncementDto(
            id = -1,
            title = null,
            content = null,
            dateTime = null
        )
    }
}
